scenarios/anomaly-detection/build.gradle

```diff
@@ -12,27 +12,23 @@ plugins {
     id 'com.github.johnrengelman.shadow' version '1.2.4'
 }
 
-repositories {
-    mavenLocal()
-    jcenter()
-    mavenCentral()
-}
-
 apply plugin: "java"
+apply plugin: "scala"
 apply plugin: "distribution"
-apply plugin: 'application'
+apply plugin: 'idea'
+apply plugin: 'eclipse'
 
 sourceCompatibility = "1.8"
-mainClassName = "io.pravega.anomalydetection.ApplicationMain"
-applicationDefaultJvmArgs = ["-Dlog4j.configuration=file:conf/log4j.properties"]
 
 archivesBaseName = 'pravega-flink-scenario-anomaly-detection'
 
 dependencies {
-    compile "org.slf4j:slf4j-log4j12:1.7.14"
     compile "io.pravega:pravega-connectors-flink_2.11:${flinkConnectorVersion}"
     compile "org.apache.flink:flink-streaming-java_2.11:${flinkVersion}"
     compile "org.apache.flink:flink-connector-elasticsearch5_2.11:${flinkVersion}"
+
+    compile "ch.qos.logback:logback-classic:1.1.7"
+    compile group: 'org.projectlombok', name: 'lombok', version: '1.16.18'
```

Comment:
I'm strongly opposed to introducing lombok into the samples.

Reply:
According to your suggestion, I removed the lombok dependency from the application.